control/purchase_order/admin.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+
+from control.purchase_order.models import Supplier, PurchaseOrder
+
+
+@admin.register(Supplier)
+class SupplierAdmin(admin.ModelAdmin):
+
+    list_display = ['id', 'supplier_name']
+
+
+@admin.register(PurchaseOrder)
+class PurchaseOrderAdmin(admin.ModelAdmin):
+
+    list_display = ['id', 'purchase_order_number']
```

Comment:
I guess, this is a WIP(work in progress), but some fields missing here. It would be also interesting in this case to have an `list_editable` as well.

See:
https://docs.djangoproject.com/en/2.1/ref/contrib/admin/